pennylane/_device.py

```diff
@@ -330,6 +330,9 @@ def check_validity(self, queue, observables):
                 for i in o.obs:
                     if i.name not in self.observables:
                         raise DeviceError("Observable {} not supported on device {}".format(i.name, self.short_name))
+            elif issubclass(o.__class__, Operation):
+                if o.name not in self.observables and not o.is_inverse:
```

Comment:
Observables are by definition hermitian, so `is_inverse` should not matter here anyways, no?

Reply:
That is true. Here the intention was to catch if the inverse of the operation was set for observables that are operations and observables at the same time. Please refer to the newer version of this method.